### ISSUE TITLE
Dropping Qt5 for OpenCV 5.0 -- Reduction of the build matrix

### ISSUE THREAD

**Issue by hmaarrfk** [conda-forge/opencv-feedstock#435]
### Comment:

@traversaro i wanted to have the conversation here to see if there was any blocker on your end.

FFMPEG 7.0 has been pretty good to me (7.1 seems problematic but I think I got through it....)

Qt6 has been good to me.

Thus I don't see the need to build out FFMPEG 6 and Qt6 anymore.

let me know if you see any reason to keep them around.

**Comment by traversaro** [comment]
I agree in dropping both (and look into closing the ffmpeg6 migration). Unfortunately I still need to build some of my packages with qt5, but as qt6 and qt5 are side-by-side side installable compiling opencv just with qt6 is perfectly fine for me.

**Comment by hmaarrfk** [comment]
However, OpenCV 5 just go pre-released: https://github.com/opencv/opencv/releases/tag/5.0.0-alpha

So we should be able to cleanly drop qt5 with that one!

**Comment by hmaarrfk** [comment]
dropped in 4.11.0
